frontend/src/pages/modelServing/__tests__/modelPod.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { checkModelStatus } from '~/pages/modelServing/screens/global/utils';
+import { mockUnscheduledPodK8sResource, mockPodK8sResource } from '~/__mocks__/mockPodK8sResource';
+
+const mockModelStatus = (status: boolean) => ({
+  FailedToSchedule: status,
```

Comment:
```suggestion
  failedToSchedule: status,
```
make this in camel case